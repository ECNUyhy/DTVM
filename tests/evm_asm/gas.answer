00000000000000000000000000000000000000000000000000000000000F423E