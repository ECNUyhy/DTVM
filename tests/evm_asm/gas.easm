// We set gas to 1000000UL in the test, 0xF423E = 1000000-2 = 999998
GAS
PUSH1 0x00
MSTORE
PUSH1 0x20
PUSH1 0x00
RETURN
