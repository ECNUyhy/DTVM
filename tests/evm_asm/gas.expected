status: success
error_code: 0
stack: []
memory: 00000000000000000000000000000000000000000000000000000000000F423E
storage: {}
transient_storage: {}
return: 00000000000000000000000000000000000000000000000000000000000F423E
events: []
